thingif/src/main/java/com/kii/thingif/ThingIFAPI.java

```diff
@@ -250,6 +250,13 @@ public ThingIFAPI build() {
          * @param alias Alias to register
          * @param actionClass List of Action subclasses
          * @return builder instance for chaining call.
+         * @throws IllegalArgumentException This exception is thrown if one or
+         * more following conditions are met.
+         * <ul>
+         *  <li>actionClass is non static inner class</li>
```

Comment:
`actionClass is a class excepting non nested class and static inner class."

We should refuse anonymous class too so instead of listing rejecting class, I prefer to list accepting class :)